.github/workflows/main.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push]
+
+jobs:
+
+  build:
+    name: Google Cloud Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    # - run:
+    - name: Authenticate w/ Google Cloud
+      uses: actions/gcloud/auth@master
+      env:
+        GCLOUD_AUTH: ${{ secrets.GCLOUD_AUTH }}
+    - name: Build, Test, Push to GCR
+      uses: actions/gcloud/cli@master
+      with:
+        entrypoint: bash
+        args: |
+          -l -c "BRANCH_NAME=$GITHUB_REPOSITORY/$(expr $GITHUB_REF : '.*/\(.*\)') && \
+          BRANCH_NAME=${BRANCH_NAME,,} && \
+          gcloud builds submit . --config cloudbuild.yaml --project zealous-zebra --substitutions BRANCH_NAME=$BRANCH_NAME"
```

Comment:
zealous-zebra is an internal gcloud identifier right?

Reply:
Yep, that's the name of the gcloud project that the service users and keys are specific to.